server/content-model/events.js

```diff
@@ -108,7 +108,8 @@ export type Event = {|
   bookingType: ?string,
   schedule?: Array<Event>,
   eventbriteId?: string,
-  isCompletelySoldOut?: boolean
+  isCompletelySoldOut?: boolean,
+  backgroundTexture?: string
```

Comment:
This shouldn't go on the event model, but rather near it - let's try to type that out...